recipes/multiaddr/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "multiaddr" %}
+{% set version = "0.0.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30b2695189edc3d5b90f1c303abb8f02d963a3a4edf2e7178b975eb417ab0ecf
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
```

Comment:
Do you need pytest-runner for pip to install?

Reply:
This is because 'setup_requires' is `setup_requires=[
        'pytest-runner',
    ],`

Reply:
Can you check? Often this is incorrect

Reply:
if it is defined in the ``setup_requires`` you do need it in host section

Reply:
otherwise ``setuptools`` will try to download it and it will fail as conda build disabled the download of packages using pip

Reply:
You could add `--no-deps` to the pip install line

Reply:
I tried this locally with ` {{ PYTHON }} -m pip install -vv --no-deps .` but it didn't seem to have any effect.